virttest/utils_test/libvirt.py

```diff
@@ -3619,6 +3619,26 @@ def check_qemu_cmd_line(content, err_ignore=False):
     return True
 
 
+def check_cmd_output(cmd, content, err_ignore=False):
+    """
+    Check the specified content in the output of the cmd
+    :param cmd: the cmd wants to check
+    :param content: the desired string to search
+    :param err_ignore: True to return False when fail
+                       False to raise exception when fail
+    :return: True if exist, False otherwise
+    """
+    if cmd is None:
+        raise exceptions.TestFail("cmd can not be None")
+    cmd_output = results_stdout_52lts(process.run(cmd, shell=True)).strip()
```

Comment:
process.run(cmd, shell=True) will raise an exception anyway if fail. Do you want to pass 'err_ignore' to process.run()?

Reply:
updated.